.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "rules"]
 	path = rules
-	url = ../capa-rules.git
+	url = ../../mandiant/capa-rules.git
 [submodule "tests/data"]
 	path = tests/data
-	url = ../capa-testfiles.git
+	url = ../../mandiant/capa-testfiles.git
```

Comment:
sorry, unrelated change:

when a user forks capa, git looks for the submodules to be found under the users account, rather than at github.com/mandiant/capa-testfiles (for example). this change makes git look to mandiant/capa-testfiles (for example) by default.

Reply:
Thanks, this confused some time ago